src/common/runtime/standalone.ts

```diff
@@ -701,6 +701,15 @@ void (async () => {
     void navigator.clipboard.writeText(logger.asJSON(2));
   });
 
+  document.getElementById('saveResultsJSON')!.addEventListener('click', () => {
+    const text = JSON.stringify(logger.asJSON(2));
```

Comment:
logger.asJSON returns JSON (a string). 

```suggestion
    const text = logger.asJSON(2);
```

Reply:
Ah sounds good. Its nice to have a real js programmer here.